medical_practitioner/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# Copyright 2017 Creu Blanca
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    'name': 'Medical Practitioner',
+    'version': '10.0.1.0.0',
+    'summary': 'Defines medical practioners',
+    'author': 'Creu Blanca',
+    'maintainer': 'Eficent, Creu Blanca, Odoo Community Association (OCA)',
```

Comment:
This should be the `author` key. `maintainer` key is a thing, but not used in OCA AFAIK

Also - I see a lot of LasLabs copyrights and credits left here. I don't recognize any of this code, however. Did LasLabs actually have a hand in any of this? If so, we should share author credit as well. Otherwise, please remove our credits.